### PR TITLE
add Ambulance model, test level and ackmann path request script

### PR DESCRIPTION
**Pull request by ddengster**
### Implemented feature

- Add ambulance model simulating a nonholonomic vehicle
- Add test level `test_ackman.building,yaml` and launch file for test level `test_ackman.launch.xml`
- Add python script `request_ackmann_path.py` to `rmf_demos_tasks`. Requires upstream PR ??(tbd)
Usage: `ros2 run rmf_demos_tasks request_ackmann_path -p <package name> -n <maps/'mapname'/nav_graphs/0.yaml> -s <Start Label> -e <End Label> -i <task id>`

### CODE REVIEW

File: rmf_demos_assets/models/Ambulance/model.sdf [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+          <box>
+            <size>0.65 1.5 1.0</size>
+          </box>
+        </geometry>
+        <material>
+          <ambient>0.0 0.0 1.0 1</ambient>
+          <diffuse>0.0 0.0 1.0 1</diffuse>
+          <specular>0.0 0.0 1.0 1</specular>
+        </material>
+      </visual>-->
```
Comment (luca-della-vedova) [inline rmf_demos_assets/models/Ambulance/model.sdf:90]: Can be removed

Comment (ddengster) [reply rmf_demos_assets/models/Ambulance/model.sdf]: removed 382aa43

**Review by luca-della-vedova — COMMENTED** [review]
Looks good to me, I think we can merge after two minor tunings (and the [upstream PR in rmf_simulation](https://github.com/open-rmf/rmf_simulation/pull/33#pullrequestreview-730506204) is merged):

1) A sanity check on the `model.sdf` physical parameters to make sure they are reasonably realistic for an ambulance
2) The test map should probably be tuned, now that the scale of the ambulance has been fixed the model barely fits in the building:
image

Other than that I think this is good to go!

**Comment by ddengster** [comment]
> The test map should probably be tuned, now that the scale of the ambulance has been fixed the model barely fits in the building:

Removed it for now. We'll toss in another test map down the line

**Review by luca-della-vedova — APPROVED** [review]

**Pull request merged**